File: StxInsider/app/src/main/java/com/stxnext/stxinsider/model/SliderItem.java
package com.stxnext.stxinsider.model;

/**
 * Created by bkosarzycki on 01.02.16.
 */
public class SliderItem {

    String header;
    String imagePath;
    String description;
    Category category;

    public String getHeader() {
        return header;
    }

    public void setHeader(String header) {
        this.header = header;
    }

    public String getImagePath() {
        return imagePath;
    }

    public void setImagePath(String imagePath) {
        this.imagePath = imagePath;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Category getCategory() {
        return category;
    }

    public void setCategory(Category category) {
        this.category = category;
    }

    public SliderItem header(String header) {
        this.header = header;
        return this;
    }

    public SliderItem imagePath(String imagePath) {
        this.imagePath = imagePath;
        return this;
    }

    public SliderItem description(String description) {
        this.description = description;
        return this;
    }


    public SliderItem category(Category category) {
        this.category = category;
        return this;
    }
}

File: StxInsider/app/src/main/kotlin/com/stxnext/stxinsider/model/SliderItem.kt
package com.stxnext.stxinsider.model

/**
 * Created by bkosarzycki on 01.02.16.
 */
class SliderItem {

    var header: String? = null
    var imagePath: String? = null
    var description: String? = null
    var category: Category? = null
    var imageResource: Int? = null

    fun header(header: String): SliderItem {
        this.header = header
        return this
    }

    fun imagePath(imagePath: String): SliderItem {
        this.imagePath = imagePath
        return this
    }

    fun description(description: String): SliderItem {
        this.description = description
        return this
    }

    fun category(category: Category): SliderItem {
        this.category = category
        return this
    }

    fun imageResource(imageResource: Int): SliderItem {
        this.imageResource = imageResource
        return this
    }
}
